<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiKinect TCPIP Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TCP/IP Client</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_Port">
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_Disconnect">
        <property name="text">
         <string>Disconnnect</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_Message">
        <property name="text">
         <string>Hello World! :)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_Send">
        <property name="text">
         <string>Send Message</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="QLabel" name="label_ConnectionState">
        <property name="text">
         <string>TCP/IP State</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_Connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPlainTextEdit" name="plainTextEdit_sent"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPlainTextEdit" name="plainTextEdit_received"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="text">
         <string>0.0.0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Kinect 2 Handler</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_Disconnect_Devices">
        <property name="text">
         <string>Disconnect Kinects</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Cpu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OpenGL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OpenCL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cuda</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_Grab_Devices">
        <property name="text">
         <string>Grab All</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_Register">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_Connect_Devices">
        <property name="text">
         <string>Connect Kinects</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_savePC">
        <property name="text">
         <string>Save PC</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_Save_Settings">
        <property name="text">
         <string>Save Settings</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_GrabAndTransmit">
        <property name="text">
         <string>GrabAndTransmit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ExpertMode">
     <property name="title">
      <string>SSH Mode (Expert only)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="Layout_ExpertMode" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_SSHUpdate">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_SSHClientCompile">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Compile</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_SSHReboot">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Reboot</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLabel" name="label_State">
           <property name="text">
            <string>State</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QPlainTextEdit" name="plainTextEdit_SSHLog">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>pushButton_SSHClientCompile</zorder>
     <zorder>Layout_ExpertMode</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_sent</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_received</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_ExpertMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>Layout_ExpertMode</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
